modules/nw-mutual-tls-auth.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * ingress/ingress-operator.adoc
+
+[id=nw-mutual-tls-auth_{context}]
+= Configuring Mutual TLS Authentication
```

Comment:
Apologies for missing this earlier, please use sentence case in [headings](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings).